### PR TITLE
WIP: Adding verified page.

### PR DESCRIPTION
**Pull request by Anish-Agnihotri**
##### Description

PR that aims to add a page to explain the upcoming verified badges and offer a quick access button to apply for one.

##### Checklist

- [X] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [X] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [ ] Linter status: 100% pass
- [X] Changes don't break existing behavior
- [X] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [ ] Completed: Code review by core team and any requested changes

##### Affected core subsystem(s)

User Interface + Design.

##### Refers/Fixes

Fixes parts of #3793 and #2510.

##### Testing and Sign-off

Tested across two devices and at multiple resolutions.

###### Contributor

- [X] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [X] Tested all changes **locally**
- [X] Verified existing functionality
- [X] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging or local deployment

### CODE REVIEW

**Comment by Anish-Agnihotri** [comment]
**TODO:**

- [ ] Figure out how to get image working in header.
- [ ] Get content from @willsputra including FAQ information + typeform link.

**Comment by PixelantDesign** [comment]
Could you provide a screenshot @Anish-Agnihotri ?

**Comment by PixelantDesign** [comment]
Could we review this @octavioamu @SaptakS ?

**Comment by Anish-Agnihotri** [comment]
Here is a screenshot: 

verified_page_gitcoin

**Comment by PixelantDesign** [comment]
Looks good could we get this merged @Anish-Agnihotri @octavioamu @SaptakS ?

**Comment by Anish-Agnihotri** [comment]
@PixelantDesign Think it still needs to get the typeform link and the information for the FAQ before we can merge, since it is just dummy content for the time being.

File: app/assets/v2/css/verified.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    }
+    .verified-cta > div:last-child > div {
+        width: 50%;
+        margin: 0 auto;
+    }
+    .verified-cta > div > img {
+        width: 200px;
+        float: none;
+    }
+}
```
Comment (thelostone-mc) [inline app/assets/v2/css/verified.css]: Can we stick to 1tab == 2 spaces ?

**Comment by thelostone-mc** [comment]
@octavioamu / @Anish-Agnihotri would it make more sense to replace the png with an svg ?

File: app/retail/templates/verified.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            <a href="" class="btn btn-gc-blue m-2 rounded">
+                <span class="pl-1">Request Verification</span>
+            </a>
+        </div>
+        <div class="row pb-5">
+            <div class="container">
+                <h3 class="my-5">FAQ</h3>
+
+                <div class="mb-3 faq-item">
+                    <h5 class="accordion">Lorem ipsum dolor sit amet, consectetur adipiscing elit</h5>
```
Comment (owocki) [inline app/retail/templates/verified.html]: @PixelantDesign we need copy for this!

**Comment by Anish-Agnihotri** [comment]
Would definitely make more sense. Especially considering we can then optimize the SVG file size. I’ll export the Sketch file to SVG, and then recommit tonight.

**Comment by PixelantDesign** [comment]
Sent this to @octavioamu who is adding this in the FAQ

1.
What does verified mean?

A verified contributor has met criteria that a Gitcoin core team member deems as a trusted contributor.

2.
Does verified contributor mean that I should use this contributor on my project?

A verified contributor is someone who has worked with, interacted with or been recommended by a fellow funder. When looking for the right contributor for your projects, please use the vetting criteria you would normally use.

3.
How can I become verified?

Complete the typeform link <here> to send a request to the Gitcoin Core team.

4.
What happens after I request verification?

An email will be sent to the Gitcoin Core team for review. Please expect 72 hours for a verified confirmation.

5.
What are the perks of being a verified contributor?

Being verified is a great way to let potential funders  or repo owners know that you are a dependable and quality contributor. This may allow for opportunities on new and exciting open source projects, bounties, or grants!

**Comment by PixelantDesign** [comment]
We'll take it from here @Anish-Agnihotri thanks for your help!

**Comment by Anish-Agnihotri** [comment]
Much appreciated! Looking forward to seeing where it goes.

Please do let me know if any help is needed and I'd be happy to clean stuff up and lend a hand. 

Cheers,

Anish

**Comment by octavioamu** [comment]
Closed by #3911

**Pull request closed without merging**